docs/notebook/import_data.rst

```diff
@@ -48,3 +48,19 @@ For Imviz::
     imviz = Imviz()
     imviz.app
     imviz.load_data("/path/to/data/image.fits")
+
+
+Importing Custom Line Lists
+---------------------------
+Jdaviz comes with curated line lists built from the scientific community. If you can't find the lines you need, you can add your own!::
+
+    from astropy.table import QTable
+    import astropy.units as u
+
+    lt = QTable()
+    lt['linename'] = ['Hbeta','Halpha']
```

Comment:
This is not the only way to construct a table. Maybe link to https://docs.astropy.org/en/latest/table/construct_table.html using intersphinx?

If you foresee people building a huge line list table, also link to https://docs.astropy.org/en/latest/table/index.html#performance-tips (if not, you don't have to). I'm talking about like maybe several thousand rows.

Reply:
Always a fan of additional documentation! I'll add those links, thanks!

Reply:
@pllim I never used intersphinx before (and it looks like there are a few places where links are still hardcoded). Any help to try to link to those pages would be much appreciated! `#newbie` :P

Reply:
Awesome! Much thanks! I got the Construct Table working, but I'm running into issues with the performance link. I found the `.. _astropy-table-performance:` header at the top of https://raw.githubusercontent.com/astropy/astropy/main/docs/table/performance.inc.rst, so I tried linking it using 
```
:ref:`<astropy:astropy-table-performance>`
```
but that didn't generate a link, and just generated raw text as "<astropy:astropy-table-performance>". Any ideas where I went wrong?

Reply:
I think there's essentially a 0% chance anyone will be adding so many lines that they'll need the table performance link, if it's not working then just remove it IMO.

Reply:
Works for me :)